docs/wrapper.asciidoc

```diff
@@ -0,0 +1,55 @@
+[[wrapper-support]]
+=== Wrapper Support
+
+experimental::[]
+
+The following frameworks are supported using our new wrapper script for no-code-
+changes instrumentation:
+
+ * Flask
+
+Please keep in mind that these instrumentations are a work in progress! We'd
+love to have feedback on our
+https://github.com/elastic/apm-agent-python/issues/new/choose[issue tracker].
+
+[[wrapper-usage]]
+==== Usage
+
+When installing the agent, an entrypoint script, `elasticapm-run` should be
+installed as well. You can use this script to instrument your app (assuming it's using a supported framework) without changing your code!
+
+[source,bash]
+----
+$ elasticapm-run --version
+elasticapm-run 6.14.0
+----
+
+Alternatively, you can run the entrypoint directly:
+
+[source,bash]
+----
+$ python -m elasticapm.instrumentation.wrapper.__init__ --version
```

Comment:
Maybe we can get rid of the somewhat ugly `__init__`. From the docs of the `-m` flag:

> Package names (including namespace packages) are also permitted. When a package name is supplied instead of a normal module, the interpreter will execute `<pkg>.__main__` as the main module. This behaviour is deliberately similar to the handling of directories and zipfiles that are passed to the interpreter as the script argument.

So could we add a `__main__` module that calls `setup()`...?

(I didn't try this)